docs/internal-documentation/distrib-testing/Debian-build-environment.md

```diff
@@ -24,7 +17,11 @@ This instruction is for creating it on Debian. See other files for other OSs.
 3. Install prerequisites
 
    ```sh
-   sudo apt-get install cpanminus gettext
+   sudo apt-get install git cpanminus gettext autoconf automake build-essential libdevel-checklib-perl libmodule-install-xsutil-perl libssl-dev libidn2-dev libldns-dev libtool
```

Comment:
libldns-dev should not be needed unless you use ldns shared library (not "internal" ldns)

Reply:
Updated